testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHivePropertiesTable.java

```diff
@@ -0,0 +1,94 @@
+
+public class TestHivePropertiesTable
+        extends HiveProductTest
+{
+    @Test
+    public void testTrinoViewPropertiesTable()
+            throws Exception
+    {
+        onTrino().executeQuery("DROP TABLE IF EXISTS test_trino_view_properties_base");
+        onTrino().executeQuery("DROP VIEW IF EXISTS test_trino_view_properties");
+        onTrino().executeQuery("CREATE TABLE test_trino_view_properties_base (col INT)");
+        onTrino().executeQuery("CREATE VIEW test_trino_view_properties AS SELECT * FROM test_trino_view_properties_base");
+
+        assertThat(query("SHOW COLUMNS FROM \"test_trino_view_properties$properties\""))
+                .containsExactlyInOrder(
+                        row("comment", "varchar", "", ""),
+                        row("presto_query_id", "varchar", "", ""),
+                        row("presto_version", "varchar", "", ""),
+                        row("presto_view", "varchar", "", ""),
+                        row("transient_lastddltime", "varchar", "", ""),
+                        row("trino_created_by", "varchar", "", ""));
+
+        assertThat(query("SELECT * FROM \"test_trino_view_properties$properties\""))
+                .hasRowsCount(1)
+                .containsExactlyInOrder(new Row(getTablePropertiesOnHive("test_trino_view_properties")));
+
+        onTrino().executeQuery("DROP VIEW IF EXISTS test_trino_view_properties");
+        onTrino().executeQuery("DROP TABLE IF EXISTS test_trino_view_properties_base");
+    }
+
+    @Test(groups = HIVE_VIEWS)
+    public void testHiveViewPropertiesTable()
+            throws Exception
+    {
+        onTrino().executeQuery("DROP TABLE IF EXISTS test_hive_view_properties_base");
+        onTrino().executeQuery("DROP VIEW IF EXISTS test_hive_view_properties");
+        onTrino().executeQuery("CREATE TABLE test_hive_view_properties_base (col INT)");
+        onHive().executeQuery("CREATE VIEW test_hive_view_properties AS SELECT * FROM test_hive_view_properties_base");
+
+        // Use "contains" method because the table properties for Hive views aren't identical among testing environments
+        assertThat(query("SHOW COLUMNS FROM \"test_hive_view_properties$properties\""))
+                .contains(row("transient_lastddltime", "varchar", "", ""));
+
+        assertThat(query("SELECT * FROM \"test_hive_view_properties$properties\""))
+                .hasRowsCount(1)
+                .containsExactlyInOrder(new Row(getTablePropertiesOnHive("test_hive_view_properties")));
+
+        onTrino().executeQuery("DROP VIEW IF EXISTS test_hive_view_properties");
+        onTrino().executeQuery("DROP TABLE IF EXISTS test_hive_view_properties_base");
+    }
+
+    private static List<String> getTablePropertiesOnHive(String tableName)
+            throws SQLException
+    {
+        ImmutableList.Builder<String> properties = ImmutableList.builder();
+        try (Statement statement = onHive().getConnection().createStatement();
```

Comment:
`onHive().executeQuery("SHOW TBLPROPERTIES " + tableName).rows()`?

Reply:
Thanks, changed to use the suggested method.